test/tests_for_utils/catalog_cache_test.py

```diff
@@ -0,0 +1,71 @@
+# This test only tests code that can be exercised without a network connection to services.
+# That code is tested in integration tests.
+import pytest
+
+from unittest.mock import create_autospec, MagicMock
+
+from installed_clients.CatalogClient import Catalog
+from execution_engine2.utils.catalog_cache import CatalogCache
+
+
+@pytest.fixture
+def catalog():
+    catalog = create_autospec(Catalog, spec_set=True, instance=True)
+    catalog.list_client_group_configs = MagicMock(
+        return_value={
+            "module_name": "module_name",
+            "function_name": "function_name",
+            "client_groups": "client_groups",
+        }
+    )
```

Comment:
I don't think you need the MagicMock, you can just do
```
catalog.list_client_group_configs.return_value = {...}
```

Reply:
ok changed

Reply:
:+1: